connectors/twilio/element-templates/twilio-connector.json

```
@@ -0,0 +1,548 @@
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "accountSid"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authType",
+        "oneOf": [
+          "apiKeySecret",
+          "accountSidAuthToken"
+        ]
+      }
+    },
+    {
+      "value": "basic",
+      "type": "Hidden",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "post",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "sendSms"
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "get",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "listMessages",
+          "getMessage"
+        ]
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "={\"content-type\":\"application/x-www-form-urlencoded\"}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "headers"
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "sendSms"
+      },
+      "optional": false
+    },
+    {
+      "label": "API key",
+      "description": "Your Twilio API key. See <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio/\" target=\"_blank\">documentation</a> for details",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.username"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authType",
+        "equals": "apiKeySecret"
+      }
+    },
+    {
+      "label": "API secret",
+      "description": "Your Twilio API secret. See <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio/\" target=\"_blank\">documentation</a> for details",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.password"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authType",
+        "equals": "apiKeySecret"
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "=accountSid",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.username"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authType",
+        "oneOf": [
+          "accountSidAuthToken"
+        ]
+      }
+    },
+    {
+      "label": "Auth token",
```

1. Let's not use jargon `Auth` => `Authentication` etc.
2. Let's maintain the same style of labels everywhere. If you write here `Authentication token` then the next one has to be `Message text` and not `...Text` with capitalized first letter.

```suggestion
      "label": "Authentication token",
```